pages/QuizSelection.tsx

```diff
@@ -1,15 +1,32 @@
 import React from 'react';
+import QuizCard from '../components/sections/QuizCard';
+import styles from '../styles/QuizSelection.module.scss';
+import Link from 'next/Link';
+import ArtAnimation from '../components/sections/ArtAnimation';
 
+const imageUrls: [string, string, string, string] = [
```

Comment:
I know this array is for testing purposes but again as a tip, you can use string[] for an array of the same type.

Also for testing purposes, I'd consider holding these images in the state since they will be fetched and the page has to re-render to show these changes.
```suggestion
const imageUrls: string[] = [
```